### PR TITLE
KDevelop: make it work

### PR DESCRIPTION
**Pull request by extrowerk**
#sudo hp kdevelop

### CODE REVIEW

File: dev-util/kdevelop/kdevelop-5.2.1.recipe [hunk 1]
```diff
@@ -123,8 +163,10 @@ BUILD_REQUIRES="
 	devel:libQt5Svg$secondaryArchSuffix
 	devel:libQt5Test$secondaryArchSuffix
 	devel:libQt5WebKit$secondaryArchSuffix
+	devel:libQt5WebKitWidgets$secondaryArchSuffix
 	devel:libQt5Widgets$secondaryArchSuffix
 	devel:libQt5Xml$secondaryArchSuffix
+
```
Comment (diversys) [inline dev-util/kdevelop/kdevelop-5.2.1.recipe]: Extra new line

**Comment by diversys** [comment]
LGTM

File: dev-util/kdevelop/patches/kdevelop-5.2.1.patchset [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+From: =?UTF-8?q?Zolt=C3=A1n=20Mizsei?= <zmizsei@extrowerk.com>
+Date: Tue, 3 Apr 2018 20:08:02 +0200
+Subject: Haiku supporting patches
+
+
+diff --git a/kdevplatform/util/dbus_socket_transformer/CMakeLists.txt b/kdevplatform/util/dbus_socket_transformer/CMakeLists.txt
+index f9ce6ac..cfebf7b 100644
+--- a/kdevplatform/util/dbus_socket_transformer/CMakeLists.txt
++++ b/kdevplatform/util/dbus_socket_transformer/CMakeLists.txt
+@@ -15,5 +15,10 @@ if(HAVE_MSG_NOSIGNAL)
```
Comment (fbrosson) [inline dev-util/kdevelop/patches/kdevelop-5.2.1.patchset]: If you added this line
```
+# explicit linking to Haiku's separate libnetwork needed due to socket() usage
```
in the patchset, the `@@ -15,5 +15,10 @@` won't be correct. You need to apply the patch without the newly added comment, then add the comment to `kdevplatform/util/dbus_socket_transformer/CMakeLists.txt` and, finally, regenerate the patchset.
Well, you could, instead, edit the `-15,5 +15,10` manually (if you speak `diff` fluently), but the `cfebf7b` in `index f9ce6ac..cfebf7b` would not be _correct_.

Comment (extrowerk) [reply dev-util/kdevelop/patches/kdevelop-5.2.1.patchset]: I knew you will spot it :)

**Pull request merged**